js/state/frameStateUtil.js

```diff
@@ -349,6 +349,34 @@ function undoCloseFrame (windowState, closedFrames) {
  * @return Immutable top level application state ready to merge back in
  */
 function removeFrame (frames, closedFrames, frameProps, activeFrameKey) {
+  function getNewActiveFrame (activeFrameIndex) {
```

Comment:
just a nit, but I think for readability it would be better add `frames` as a parameter and move this out of `removeFrame`